modules/3d-tiles/src/tiles-3d-loader.js

```diff
@@ -36,6 +36,7 @@ function getBaseUri(tileset) {
 async function parseTile(arrayBuffer, options, context) {
   const tile = {};
   tile.content = tile.content || {};
+  tile.segmentationData = tile.segmentationData || null;
```

Comment:
Tile is being initialized here?
```suggestion
  const tile = {
     content: {},
     segmentationData: null
  };
```

Reply:
Fixed